docs/external_system.md

```diff
@@ -250,110 +275,70 @@ steps needed to configure SAT to use externally-accessible API endpoints exposed
-|`sat slscheck`|Requires authentication to the API gateway.|`sat-slscheck`|Perform a cross-check between SLS and HSM.|
-|`sat status`|Requires authentication to the API gateway.|`sat-status`|Report node status across the HPE Cray EX system.|
-|`sat swap`|Requires authentication to the API gateway.|`sat-swap`|Prepare HSN switch or cable for replacement and bring HSN switch or cable into service.|
-|`sat xname2nid`|Requires authentication to the API gateway.|`sat-xname2nid`|Translate node and node BMC XNames to node IDs.|
-|`sat switch`|**This command has been deprecated.** It has been replaced by `sat swap`.|
-
-In order to authenticate to the API gateway, you must run the `sat auth` command. This command will prompt for a password
-on the command line. The username value is obtained from the following locations, in order of higher precedence to lower
-precedence:
-
-- The `--username` global command-line option.
-- The `username` option in the `api_gateway` section of the config file at `~/.config/sat/sat.toml`.
-- The name of currently logged in user running the `sat` command.
-
-If credentials are entered correctly when prompted by `sat auth`, a token file will be obtained and saved to
-`~/.config/sat/tokens`. Subsequent sat commands will determine the username the same way as `sat auth` described above,
-and will use the token for that username if it has been obtained and saved by `sat auth`.
+Initially, as part of the installation and configuration, SAT authentication is
+set up so SAT commands can be used in later steps of the install process. The
+admin account used to authenticate with `sat auth` must be enabled in Keycloak
+and must have its *assigned role* set to *admin*. For more information on
+editing *Role Mappings*, see *Create Internal User Accounts in the Keycloak
+Shasta Realm* in the [Cray System Management Documentation](https://cray-hpe.github.io/docs-csm/).
+For more information on authentication types and authentication credentials,
+see [SAT Command Authentication](introduction.md#sat-command-authentication).
 
 ### Prerequisites
 
-- The `sat` CLI has been installed following [Install and Configure SAT](#install-and-configure-sat).
+- The `sat` CLI has been installed following [Install and Configure
+  SAT](#install-and-configure-sat).
 
 ### Procedure
 
-The following is the procedure to globally configure the username used by SAT and authenticate to the API gateway:
+The following is the procedure to globally configure the username used by SAT
+and authenticate to the API gateway.
 
 1. Generate a default SAT configuration file, if one does not exist.
 
    ```screen
-   ncn-m001# sat init
+   $ sat init
    Configuration file "/root/.config/sat/sat.toml" generated.
```

Comment:
This line will have a different path, depending on the user's home directory on the system. E.g., on my macbook, it will be something like:

```
Configuration file "/Users/haasken/.config/sat/sat.toml" generated.
```

Perhaps we could do something like this, which isn't exactly as it appears, but should get the point across:

```suggestion
Configuration file "$HOME/.config/sat/sat.toml" generated.
```

Reply:
Updated in next commit